.github/workflows/haskell-ci-hackage.patch

```diff
@@ -45,12 +45,12 @@ set in GitHub repository secrets.
            - compiler: ghc-8.10.7
              compilerKind: ghc
              compilerVersion: 8.10.7
-@@ -209,8 +217,80 @@
+@@ -240,8 +240,80 @@
            ${CABAL} -vnormal check
        - name: haddock
          run: |
--          $CABAL v2-haddock $ARG_COMPILER --with-haddock $HADDOCK $ARG_TESTS $ARG_BENCH all
-+          $CABAL v2-haddock $ARG_COMPILER --with-haddock $HADDOCK $ARG_TESTS $ARG_BENCH --haddock-for-hackage --builddir $GITHUB_WORKSPACE/haddock all
+-          $CABAL v2-haddock --haddock-all $ARG_COMPILER --with-haddock $HADDOCK $ARG_TESTS $ARG_BENCH all
++          $CABAL v2-haddock --haddock-all $ARG_COMPILER --with-haddock $HADDOCK $ARG_TESTS $ARG_BENCH --haddock-for-hackage --builddir $GITHUB_WORKSPACE/haddock all
```

Comment:
This breaks the docs tarball that would be uploaded to Hackage on release :-(

Reply:
Oh.  These were just regenerated with `haskell-ci regenerate`; I guess we need another patch

Reply:
Fixed by https://github.com/xmonad/xmonad-contrib/pull/775/commits/688041791176f0b6758012e35a255f9836486629, sorry for the panic :-)